http/src/test/resources/reference.conf

```diff
@@ -37,7 +37,7 @@ mysql {
   port = 3311
   db {
     driver = "com.mysql.cj.jdbc.Driver"
-    url = "jdbc:mysql://"${mysql.host}":"${mysql.port}"/leotestdb?createDatabaseIfNotExist=true&useSSL=false&rewriteBatchedStatements=true&nullNamePatternMatchesAll=true"
+    url = "jdbc:mysql://"${mysql.host}":"${mysql.port}"/leotestdb?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&useSSL=false&rewriteBatchedStatements=true&nullNamePatternMatchesAll=true"
```

Comment:
I would love to enable SSL everywhere and disable `allowPublicKeyRetrieval` but that seems like a project for another PR.

Reply:
Yeah, I checked that this is only used for our unit testing at least